Jetcaster/wear/src/main/java/com/example/jetcaster/theme/Color.kt

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2021 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.example.jetcaster.theme
+
+import androidx.compose.ui.graphics.Color
+import androidx.wear.compose.material.Colors
+
+internal val wearColorPalette: Colors = Colors(
```

Comment:
Can you add a comment here on why the wear color palette doesn't use the colors in the `designsystems` module?

```
// TODO: Update colors used to use colors provided in the designsystem module when the wear 
// module is upgraded to use material3
```

Reply:
done

Reply:
using colors from design system now